src/model/document.js

```diff
@@ -313,6 +313,9 @@ export default class Document {
 				this.fire( 'change', writer.batch );
 			}
 
+			// Refresh selection attributes according to the final position in the model after the change.
```

Comment:
As I understand, the selection attributes are now not refreshed on `change` at all. Does it mean that the selection might have wrong attributes after last post-fixer changes it?

Reply:
Maybe this should be added as a `change` event callback. First because of that reason, second because I think it is better if there's less selection-related code directly in `Document` class. We already have a lot of listeners in `DocumentSelection`. WDYT?

Reply:
I moved it to Document class to not split the refreshes between two classes.